### DISCUSSION TITLE
Implement Coolify.io-like support of envs and Railway.app envs (Shared, Project, Service Env)

### DISCUSSION THREAD

**Discussion by RedStar071** [Dokploy/dokploy#419] [Ideas]
is it possible to do something like this on dokply? like like another hosting (railway.app)

Example 1:
```
Service 1: 
// use env from another service example Postgres (DATABASE)
${{Postgres.DATABASE_URL}}

Project variables:
${Project.HELLO_WORLD}}
or
All project variables:
${Shared.FOO_BAR}}
```

example 2: 
![image](https://github.com/user-attachments/assets/4efad06c-aa49-4db4-9143-de573b2c20cc)
![image](https://github.com/user-attachments/assets/9d4d51c2-f04e-4474-b780-68ab0c9be848)

**Comment by Siumauricio** [comment]
Hi @RedStar071, Currently is not possible but looks like a good feature to implement in dokploy, let's add this to the roadmap

**Reply by Siumauricio** [reply]
Created a issue #443

**Reply by RedStar071** [reply]
Can I suggest more for dokploy?
